x-pack/plugins/reporting/server/export_types/csv/generate_csv/index.ts

```diff
@@ -36,9 +36,7 @@ interface SearchRequest {
 }
 
 export interface GenerateCsvParams {
-  jobParams: {
-    browserTimezone: string;
-  };
+  browserTimezone: string;
```

Comment:
It really confused me to have a `jobParams` field in an object that is a params object :\